Godeps/_workspace/src/k8s.io/kubernetes/test/e2e/kubectl.go

```diff
@@ -755,7 +755,7 @@ var _ = Describe("Kubectl client", func() {
 		var rcPath string
 		var nsFlag string
 		containerName := "redis-master"
-		BeforeEach(func() {
+		JustBeforeEach(func() {
```

Comment:
there are lots of things done in BeforeEach in this test... don't most of them need changing?